src/helpers/toggle-classes-on-redirect.ts

```diff
@@ -0,0 +1,25 @@
+export const toggleClassesOnRedirect = function (isLogined: boolean = true, page: string = 'main'): void {
+  const bodyElement = document.querySelector('body');
+  
+  if(bodyElement === null) {
+    return;
+  }
+
+  if (isLogined) {
+    bodyElement.classList.add('is-logined');
+  } else {
+    bodyElement.classList.remove('is-logined');
+  }
+
+  if (page === 'main') {
+    bodyElement.classList.add('mainpage');
+    if (!bodyElement.classList.contains('background')) {
+      bodyElement.classList.add('background');
+    }
+  } else {
+    bodyElement.classList.remove('mainpage');
+    if (bodyElement.classList.contains('background')) {
+      bodyElement.classList.remove('background');
+    }
+  }
+}
```

Comment:
Why not to use toggle, it's a very useful method for shifting element classes

You can read about this method here https://blog.replaybird.com/change-an-elements-class-with-javascript-js-add-remove-toggle-replace-contains/  №3 Toggle Element’s Class
```suggestion
export const toggleClassesOnRedirect = function (isLogined: boolean = true, page: string = 'main'): void {
  const bodyElement = document.querySelector('body');
  
  if(bodyElement === null) {
    return;
  }
  bodyElement.classList.toggle('is-logined', isLogined);

  const isMainPage = (page === 'main');
  bodyElement.classList.toggle('minpage', isMainPage)
  bodyElement.classList.toggle('background', isMainPage)
}
```